packages/ses/test/test-tame-faux-data-properties.js

```diff
@@ -0,0 +1,120 @@
+import test from 'ava';
+import '../index.js';
+
+import { tameFauxDataProperty as tfdp } from '../src/tame-faux-data-properties.js';
+
+const { freeze, defineProperty, getOwnPropertyDescriptor } = Object;
+
+test('unit test tameFauxDataProperty', t => {
```

Comment:
Can we get messages added to all these test assertions?

Reply:
Done